test/functional/app/api/test_ussd_api.py

```diff
@@ -236,3 +235,64 @@ def test_ussd_self_signup_wrong_pin_entry(test_client,
     resp = req("1212", test_client, user.phone, auth_username=external_auth_username)
     assert "END The new PIN does not match the one you entered." in resp
 
+
+def test_reset_pin_flow(test_client,
+                        init_database,
+                        create_organisation,
+                        authed_sempo_admin_user):
+
+    # define different session id
+    other_session_id = 'ATUid_05af06225e6163ec2dc9dc9cf8bc97aa000'
+
+    # create organisation
+    organisation = create_organisation
+    organisation.external_auth_password = config.EXTERNAL_AUTH_PASSWORD
+
+    # external auth password matching model definition of the same value.
+    # org.external_auth_username = 'admin_'+(org.name or '').lower().replace(' ', '_') /961ab9adc300_.py
```

Comment:
please remove commented code

Reply:
The comment isn't part of the code in anyway, it was there to provide context for why that external_auth_name was structured like that, in case we or someone else ever comes back to fix this, I feel the context would be helpful.
But then again it could be a less confusing comment so let me just make it clearer real quick.